### ISSUE TITLE
[Quest][Classic] NPC in escort quest "Freedom to Ruul" does not always attack

### ISSUE THREAD

**Issue by wrathofthebug** [dalaranwow/dalaran-wow#5033]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Upon starting the escort quest, the NPC just keeps walking along the escort path in Thistlefur Hold without aggroing any of the nearby hostile mobs. As a result, the player has to deal with multiple mobs alone while still trying to keep pace with the NPC which is almost impossible unless you are multiple levels above the quest level. Also, even if you manage to keep up with the NPC to the entrance of the Hold, the final wave of mobs(which the NPC finally aggros) numbers about 10 which is impossible to clear with just you and the NPC. This escort quest is supposed to be able to be done as a solo player so there is definitely something wrong with the number of mobs as well at the end. I suspect there's a glitch that "double spawns" the mobs. Either that or the final spawn mobs are already present like normal mobs before the NPC reaches and thus when the actual wave spawns it becomes a form of "double spawn".
**Expected behaviour**: Tell us what should happen instead
The NPC should be aggoring neaby hostile mobs, stopping to fight them upon aggroing. There should also be half or less the current number of mobs spawned for the final engage at the entrance to Thistlefur Hold.
**Steps to reproduce the problem**:

1. Enter Thistlefur Hold and find the Ruul Snowhoof in a cage
2. Start the escort quest
3. Observe the NPC's behaviour during escort, as well as the vast number of mobs that you have to deal with near the end of the escort path at the entrance of Thistlefur Hold

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello and thank you for the report @wrathofthebug.
Issue confirmed on PTR. Ruul will attack only the waves spawn at specifit waypoints and ignore all normal creatures in the cave.
![image](https://user-images.githubusercontent.com/6797611/32613010-65050966-c572-11e7-9e83-b547a7b0c481.png)
